etc/kayobe/kolla.yml

```diff
@@ -37,7 +37,7 @@ kolla_ansible_source_url: https://github.com/stackhpc/kolla-ansible.git
 
 # Version (branch, tag, etc.) of Kolla Ansible source code repository if type
 # is 'source'.
-kolla_ansible_source_version: refs/tags/stackhpc/7.1.1.2-rc
+kolla_ansible_source_version: refs/tags/stackhpc/7.1.1.3
```

Comment:
nit: It looks like we never had a 7.1.1.2 release - can we use that?

Reply:
Can do!